src/lib.rs

```diff
@@ -128,6 +129,12 @@ struct FtInner<Device> {
     pins: [Option<PinUse>; 8],
 }
 
+impl<Device: FtdiCommon> Drop for FtInner<Device> {
+    fn drop(&mut self) {
+        self.ft.close().unwrap();
```

Comment:
Thanks for the PR!

One suggestion: if the `init` fails due to something like somebody pulling the cable out during a transfer then `close` will fail too.  I think this `unwrap` should be replaced `ok` to avoid a panic condition in those scenarios.

```suggestion
        self.ft.close().ok();
```